pkg/plugin/trivy/plugin.go

```diff
@@ -548,6 +571,220 @@ func (p *plugin) getPodSpecForStandaloneMode(config Config, spec corev1.PodSpec,
 	}, secrets, nil
 }
 
+//FileSystem mode is quite similar to standalone mode.
+//The only difference is that instead of scanning the resource by name,
+//We scanning the resource place on a specific file system location using the following command.
+//
+//     trivy --quiet fs  --format json --ignore-unfixed  file/system/location
+//
+func (p *plugin) getPodSpecForFileSystemMode(config Config, spec corev1.PodSpec, credentials map[string]docker.Auth) (corev1.PodSpec, []*corev1.Secret, error) {
+	var secret *corev1.Secret
+	var secrets []*corev1.Secret
+	if len(credentials) > 0 {
```

Comment:
Credentials only make sense with imagePullSecrets. The idea of "FileSystem" mode is to scan images cached on K8s node, therefore we don't need imagePullSecrets to scan them.